### PR TITLE
Add chainId instead of network-url in main route.

### PR DESCRIPTION
**Pull request by calina-c**
Closes https://github.com/oceanprotocol/privateissues/issues/21.

### CODE REVIEW

**Comment by trentmc** [comment]
Ideally this points to a github issue, and the github issue describes the motivation behind doing this change. It can be short.

**Comment by calina-c** [comment]
> Ideally this points to a github issue, and the github issue describes the motivation behind doing this change. It can be short.

Added in description.

File: deployment/README.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ confirm provider service is accessible on localhost port 8030/tcp:
...
+{"computeAddress":null,"chainId":"1","providerAddress":"0xe08A1dAe983BC701D05E492DB80e0144f8f4b909","serviceEndpoints":{"computeDelete":["DELETE","/api/v1/services/compute"],"computeStart":["POST","/api/v1/services/compute"],"computeStatus":["GET","/api/v1/services/compute"],"computeStop":["PUT","/api/v1/services/compute"],"download":["GET","/api/v1/services/download"],"encrypt":["POST","/api/v1/services/encrypt"],"fileinfo":["POST","/api/v1/services/fileinfo"],"initialize":["GET","/api/v1/services/initialize"],"nonce":["GET","/api/v1/services/nonce"]},"software":"Provider","version":"0.4.8"}
```
Comment (DMats) [inline deployment/README.md:221]: The instructions above this line set `NETWORK_URL: ropsten`. For consistency, change the chainId to 3.

**Review by DMats — CHANGES_REQUESTED** [review]

**Review by DMats — CHANGES_REQUESTED** [review]
I asked a question related to this PR at https://github.com/oceanprotocol/privateissues/issues/21#issuecomment-872409680. I'm hoping to get an answer before approving.

**Review by DMats — DISMISSED** [review]

**Review by alexcos20 — CHANGES_REQUESTED** [review]
If we can store the chain_id at start  and use that value, would be great.


Vector attack:  Somebody will refresh your root endpoint 100.000 times.  this translates to 100k infura requests.  If you have a limited account, then your provider is down

**Review by alexcos20 — APPROVED** [review]

**Pull request merged**